doc/source/serve/advanced-guides/multi-app-container.md

```diff
@@ -0,0 +1,175 @@
+
+::::{tab-set}
+
+:::{tab-item} Ubuntu 20.10 and newer
+```bash
+sudo apt-get update
+sudo apt-get install podman -y
+```
+:::
+
+:::{tab-item} Ubuntu 20.04 and older
+The podman package is only available in the official repositories for Ubuntu 20.10 and newer. To install podman in Ubuntu 20.04, you need to first add the software repository as a debian source.
+
+```bash
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 4D64390375060AA4
+sudo apt-get update
+sudo apt-get install podman -y
+```
+:::
+
+:::{tab-item} Other
+See [Podman Installation Instructions](https://podman.io/docs/installation).
+:::
+
+::::
+
+## Run Serve application in a container
+
+In this example, we are going to walk through deploying two applications in separate containers: a whisper model and a Resnet50 image classification model. 
+
+First, let's set up the docker images with the required dependencies. 
+
+:::note
+The Ray version and Python version in the container should match those of the host environment exactly. Note that for Python, the versions must match down to the patch number.
+:::
+
+In `whisper.Dockerfile`:
+* We are running the Whisper model on GPU, so the base image for the Whisper Dockerfile is the latest Ray GPU image, `rayproject/ray:latest-py38-gpu`.
+* We install the package `faster_whisper`, which is a dependency for the whisper model.
+* We then download the source code for the whisper application into `whisper_example.py`.
+Similarly, `resnet.Dockerfile` uses the appropriate base image, installs necessary dependencies, and downloads the resnet model source code.
+Save the following to files named `whisper.Dockerfile` and `resnet.Dockerfile`.
+
+::::{tab-set}
+:::{tab-item} whisper.Dockerfile
+```dockerfile
+FROM rayproject/ray:latest-py38-gpu
+
+RUN pip install faster_whisper==0.10.0
+RUN sudo apt-get update && sudo apt-get install curl -y
+
+RUN curl -O https://raw.githubusercontent.com/ray-project/ray/master/doc/source/serve/doc_code/whisper_example.py
+```
+:::
+:::{tab-item} resnet.Dockerfile
+```dockerfile
+FROM rayproject/ray:latest-py38-cpu
+
+RUN torch==2.0.1 torchvision==0.15.2
+RUN sudo apt-get update && sudo apt-get install curl -y
+
+RUN curl -O https://raw.githubusercontent.com/ray-project/ray/master/doc/source/serve/doc_code/resnet50_example.py
+```
+:::
+::::
+
+Then, build the corresponding docker images and push it to your choice of docker registry.
+
+::::{tab-set}
+:::{tab-item} Whisper
+```bash
+export IMG1={your-image-repo-and-name}
+docker build -t $IMG1 -f whisper.Dockerfile .
+docker push $IMG1
+```
+:::
+:::{tab-item} Resnet
+```bash
+export IMG2={your-image-repo-and-name}
```

Comment:
I feel for people who's not familiar with building and pushing docker images might not aware `your-image-repo-and-name` for `IMG1` and `IMG2` should be different. Maybe name them `whisper_image:latest` and `restnet_image:latest` or something alone those lines.

Reply:
Yup - I added a note about running locally, and I also changed the image names to `whisper_image:latest` and `resnet_image:latest`, adding a note that the user should change the names if they need to.